book/src/tooling-check.md

```diff
@@ -30,6 +30,6 @@ Hello, world!
 
 This ensures that the whole toolchain works correctly and finds the system linker. This should work at all times, if it doesn't, immediately ask for a trainer.
 
-# QT specifics
+# Qt specifics
 
-TODO tool check for QT, compiling a hello world app.
+TODO tool check for Qt, compiling a hello world app.
```

Comment:
also ensure that `qmake` is in the PATH or set via an env variable so that CXX-Qt can find it